### PR TITLE
Replace mongodb with postgres

### PR DESCRIPTION
**Pull request by heron2014**
### PR 1
Related to #13

### CODE REVIEW

File: db/client.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+require('env2')('.env');
+
```
Comment (sohilpandya) [inline db/client.js]: Are you going to use this client for your testing, if so you may want to add an env for testing as well 

Also worth spending some time sorting out all of the tests for this in my opinion.

**Comment by sohilpandya** [comment]
Also wanted to ask if you have thought a little more about the schema, as you currently only have 1 table, chopping and changing the database in SQL is a little more of an issue than in noSQL ones like mongo/redis. 

I do remember you did have a really good diagram of the architecture for the DB. Has that changed quite a bit? (I saw it like a month ago! :( )

**Comment by heron2014** [comment]
Yes I do have a more progressive schema, but I kind of wanted to do step by step process. So first I want to implement auth and only the table related to it. Did you work in different manner?

**Comment by sohilpandya** [comment]
We tried to envisage the whole schema and how it would interact. (we also had some dummy data in the db so that it would come in handy when we ran our tests)

**Comment by heron2014** [comment]
Here is my schema: https://github.com/brainsandbrawns/documentation/blob/master/db.md

**Comment by heron2014** [comment]
Thanks Sohil. Definitely will add users to the testing schema, but first I need to inject my testing db and I have some silly issues or maybe I am now too tired. One of the two haha

**Comment by sohilpandya** [comment]
Will you be working on this branch or a separate one? I could merge this, but won't recommend doing it until you have some testing set up for it. 💯  LOL

**Comment by heron2014** [comment]
Sohil you are more than welcome to create a new branch testing and help me out :)

**Comment by heron2014** [comment]
Yeah, don't merge it yet. I will make sure it has some initial tests at least.

**Pull request merged**